lib/ui/painting.dart

```diff
@@ -1577,6 +1577,11 @@ class Codec extends NativeFieldWrapperClass2 {
   /// * -1 for infinity repetitions.
   int get repetitionCount native 'Codec_repetitionCount';
 
+  /// Clears the in-memory decoded frame cache and disables future caching of
+  /// decoded frames. This negatively impacts CPU usage, but reduces memory
+  /// consumption.
+  void clearAndDisableFrameCache() native 'Codec_clearAndDisableFrameCache';
```

Comment:
Should we have an EnableFrameCache() as well? (e.g an app shows a page with one big GIF and disables cache, and continues to a page with a small GIF so re-enables the cache)